test/test_retriever.py

```diff
@@ -215,6 +215,42 @@ def test_drop_statement():
     assert test_engine.drop_statement(
         'TABLE', 'tablename') == "DROP TABLE IF EXISTS tablename"
 
+def test_download_from_kaggle_competition():
+    """Test the downloading of dataset from kaggle, of a known competition"""
+    setup_functions()
+    files = test_engine.download_from_kaggle(
+        data_source="competition",
+        dataset_name="titanic",
+        archive_dir=raw_dir_files,
+        archive_full_path=os.path.join(raw_dir_files, "titanic")
+    )
+    assert ["gender_submission.csv",  "test.csv", "train.csv"] == files
+
+def test_download_from_kaggle_dataset():
+    """Test the downloading of dataset from kaggle, of a known dataset"""
+    setup_functions()
+    files = test_engine.download_from_kaggle(
+        data_source="dataset",
+        dataset_name="uciml/iris",
+        archive_dir=raw_dir_files,
+        archive_full_path=os.path.join(raw_dir_files, "iris")
+    )
+    assert ['Iris.csv', 'database.sqlite'] == files
+
+def test_download_from_kaggle_unknown():
+    """Test the downloading of dataset from kaggle, of a erranous dataset name"""
```

Comment:
`"""Test the downloading of a dataset from kaggle using an erroneous dataset name"""`